stdlib/turtle.pyi

```diff
@@ -141,8 +141,18 @@ if sys.version_info < (3, 13):
 _Color: TypeAlias = str | tuple[float, float, float]
 _AnyColor: TypeAlias = Any
 
-# TODO: Replace this with a TypedDict once it becomes standardized.
-_PenState: TypeAlias = dict[str, Any]
+class _PenState(TypedDict):
+    shown: bool
+    pendown: bool
+    pencolor: _Color
+    fillcolor: _Color
```

Comment:
These are always returned as strings. For example:

```
>>> import turtle
>>> turtle.pen(pencolor=(1,0,0))
>>> turtle.pen()
{'shown': True, 'pendown': True, 'pencolor': '#ff0000', ...
```

However, using `str` here is not any better, because it is not wrong to have tuples in pen state dicts:

```
>>> p = turtle.pen().copy()
>>> p['pencolor'] = (0,1,0)
>>> turtle.pen(p)
>>> turtle.pen()['pencolor']
'#00ff00'
```